packages/number-field/README.md

```diff
@@ -183,7 +214,9 @@ While `Intl.NumberFormatOptions` does support a [wide range of units](https://tc
 
 Note: the polyfilling done here is very simplistic and is triggered by supplying options that would otherwise cause the `Intl.NumberFormat()` call to throw an error. Once the unsupporting unit of `px` causes the construction of the object to throw, a back up formatter/parser pair will be created without the supplied unit data. When the `style` is set to `unit`, the `unit` value of will be adopted as the _static_ unit display. This means that neither pluralization or translation will be handled within the `<sp-number-field>` element itself. If pluralization or translation is important to the delivered interface, please be sure to handle passing those strings into to element via the `formatOptions` property reactively to the value of the element or locale of that page in question.
 
-## Minimum and maximum values
+</sp-tab-panel>
+<sp-tab value="min-max">Minimum and maximum values</sp-tab>
+<sp-tab-panel value="min-max">
```

Comment:
I had a similar question for the Decimal example. Did we want to have all of these formatting section number fields to be 400px in width? 

Looks like decimal is set to 200px, and then this min/max and the step values examples do not have a width defined.

Not a blocker, just curious on your thoughts in making them all the same or not!